lib/src/core/utils/time.dart

```diff
@@ -0,0 +1,3 @@
+String formatTime(DateTime dateTime) {
+  return '${dateTime.hour}:${dateTime.minute.toString().padLeft(2, '0')}';
+}
```

Comment:
Since this function is used only in the chat feature, might be better to put it inside of features/chat/utils

Reply:
fixed